### PR TITLE
Request optimization

### PR DESCRIPTION
**Pull request by JudeDavis1**

### CODE REVIEW

**Comment by j-koziel** [comment]
will review tomorrow 👍.
had a long day today :(

**Comment by JudeDavis1** [comment]
No problem. Remember to checkout to jude-contrib to test before merging into DEV

File: src/controllers/bibleVerseController.ts [hunk 1]
```diff
@@ -1,31 +1,56 @@
...
+            // Remove non-alpha-whitespace characters e.g. <p>hello</p> => phellop
+            const outputStr = val.text.replace(/[^a-zA-Z-.\s]/g, "").trim();
+            return outputStr;
+          });
 
-    res.json({ data: `${html}` });
+        // sending response to the user
+        const resultVerse = verse[0];
+
+        if (resultVerse == undefined) {
```
Comment (j-koziel) [inline src/controllers/bibleVerseController.ts]: undefined is a falsy value in js so this line can be refactored to just (!resultVerse)

**Review by j-koziel — COMMENTED** [review]
It all looks good, just a few things which can be changed. First when I ran npm install I got warnings about the lock file being generated with an old version of npm so you might want to check that on your end but this isn't a big deal just wanted to let you know.... the latest version of npm right now I think is 8.19.2 just to confirm
image

I put a comment about refactoring for when you checked if the result verse even exists

Other than that thank you for doing this it is really helpful and I'm glad I found someone like you to collab on this :)

File: src/controllers/bibleVerseController.ts (hunk 1 above)
Comment (JudeDavis1) [reply src/controllers/bibleVerseController.ts]: Got it, will change that now...

**Comment by JudeDavis1** [comment]
> 

No worries just updated it. I'm glad to work on a team with you as well!

**Comment by JudeDavis1** [comment]
Alright, I refactored the falsy undefined bit

**Comment by JudeDavis1** [comment]
Is it all good to merge?

**Comment by JudeDavis1** [comment]
Just called `next` again for the errors in case we forget next time.

**Comment by JudeDavis1** [comment]
Hopefully should be okay to merge now

**Comment by j-koziel** [comment]
> Hopefully should be okay to merge now 

Yup looks good, was going to mention the next function call but I wasn't entirely sure... glad you picked up on it 👌

**Pull request merged**